app/src/main/res/navigation/main_nav.xml

```diff
@@ -6,11 +6,9 @@
     android:id="@+id/main_nav_graph.xml"
     app:startDestination="@id/safeBalancesFragment">
 
-    <include app:graph="@navigation/add_safe_nav" />
```

Comment:
add_save_nav.xml can be deleted now

Reply:
This has been deleted already, if you look at the other file change in this PR.